couler/steps/tensorflow.py

```diff
@@ -0,0 +1,186 @@
+import copy
+import uuid
+
+import pyaml
+
+import couler.argo as couler
+from couler.core import utils
+
+container_template = {"name": "tensorflow", "image": "", "command": ""}
+
+pod_template = {
+    "replicas": 1,
+    "restartPolicy": "",
+    "template": {"spec": {"containers": []}},
+}
+
+manifest_template = {
+    "apiVersion": '"kubeflow.org/v1alpha2"',
```

Comment:
How do we plan to support multiple versions?

Reply:
this is good point, can we leave this for user to configure. but this would confuse end-users.

Reply:
Created a issue https://github.com/couler-proj/couler/issues/42 so that others can contribute as well if they want.